clmm/support/mock_data.py

```diff
@@ -95,6 +101,11 @@ def generate_galaxy_catalog(cluster_m, cluster_z, cluster_c, cosmo, zsrc, Delta_
         Proper distance in Mpc  at the cluster redshift.
     shapenoise : float, optional
         If set, applies Gaussian shape noise to the galaxy shapes with a width set by `shapenoise`
+    mean_e_err : float, optional
+        Mean per-component ellipticity uncertainty. Currently,
+        individual uncertainties are drawn from a uniform distribution
+        in the range [0.9,1.1]*mean_e_err. If not provided, the output
```

Comment:
Is there a reason to have this specific range here? I mean is setting this choice of 10% variation arbitrary?

Reply:
totally arbitrary. We decided we wanted any implementation to get us going, untiil we have something even remotely realistic. Happy to update to your preferred range/distribution

Reply:
This is fine for now, we can re-address this when implementing more realistic error in the future.